AmplifyPlugins/API/AWSAPICategoryPlugin/Operation/APIOperationResponse.swift

```diff
@@ -32,7 +34,11 @@ extension APIOperationResponse {
 
             let successStatusCodes = 200 ..< 300
             if !successStatusCodes.contains(statusCode) {
-                throw APIError.httpStatusError(statusCode, response)
+                if let restResponse = AWSHTTPURLResponse.init(response: response, body: responseData) {
```

Comment:
Prefer the type constructor over calling `.init` as a method

```suggestion
                if let restResponse = AWSHTTPURLResponse(response: response, body: responseData) {
```